api/v1/pod_webhook_test.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+import (
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestNewPodWebHook(t *testing.T) {
+
+	var ignoredNamespaces = []string{
+		metav1.NamespaceSystem,
+		metav1.NamespacePublic,
+	}
+	podWebHook := NewPodWebHook()
+
+	if !reflect.DeepEqual(ignoredNamespaces, podWebHook.IgnoreNamespaces) {
+		t.Errorf("NewPodWebHook() = %v, want %v", podWebHook.IgnoreNamespaces, ignoredNamespaces)
+	}
+}
```

Comment:
I think it's better to test the structure itself.

```suggestion
func TestNewPodWebHook(t *testing.T) {
	got := NewPodWebHook()
	want := &PodWebhook{
		IgnoreNamespaces: []string{
			metav1.NamespaceSystem,
			metav1.NamespacePublic,
		},
	}

	if !reflect.DeepEqual(got, want) {
		t.Errorf("NewPodWebHook() = %#v, want %#v", got, want)
	}
}
```